api/src/main/resources/jakarta/persistence/orm_3_0.xsd

```diff
@@ -13,8 +13,8 @@
 -->
 
 <!-- Jakarta Persistence API object/relational mapping file schema -->
-<xsd:schema targetNamespace="https://xmlns.jakarta.ee/xml/ns/persistence/orm" 
-  xmlns:orm="https://xmlns.jakarta.ee/xml/ns/persistence/orm" 
+<xsd:schema targetNamespace="http://jakarta.ee/xml/ns/persistence/orm" 
+  xmlns:orm="http://jakarta.ee/xml/ns/persistence/orm" 
```

Comment:
Per https://github.com/eclipse-ee4j/jakartaee-platform/blob/gh-pages/publish-xml-schemas.md
these should still be https.